### PR TITLE
added createdAt and updatedAt to subscription

### PR DESCRIPTION
**Pull request by tengis617**
added `created-at` to subscription

### CODE REVIEW

File: subscription_integration_test.go [hunk 1]
```diff
@@ -197,6 +197,10 @@ func TestSubscriptionAllFieldsWithBillingDayOfMonth(t *testing.T) {
 	if x := sub1.Descriptor.URL; x != "example.com" {
 		t.Fatalf("got descriptor url %#v, want example.com", x)
 	}
+	year, month, day := time.Now().Date()
+	if x := sub1.CreatedAt; x.Year() != day || x.Month() != month || x.Day != day {
+		t.Fatalf("got %v %v %v , want %v %v %v", x.Year(), x.Month(), x.Day(), year, month, day)
+	}
```
Comment (leighmcculloch) [inline subscription_integration_test.go]: This test could be flaky. It's best if it can just check it's not nil.

Comment (tengis617) [reply subscription_integration_test.go]: @leighmcculloch i see. Let me fix that real quick

**Comment by tengis617** [comment]
@leighmcculloch im adding `updated-at` as well

**Comment by leighmcculloch** [comment]
Thanks!!!!

**Pull request merged**